platform/android/export/gradle_export_util.h

```diff
@@ -0,0 +1,69 @@
+//
+// Created by Aman Jain on 7/6/20.
+//
```

Comment:
This should match the default file header for the Godot project.
See [here](https://github.com/godotengine/godot/blob/b5a73c9e5c9ddb45b14424fb787fea67f695a85b/platform/android/plugin/godot_plugin_jni.h#L1) for reference.

Reply:
Are the`#include` directives fine where they are or should I change them?

Reply:
The `#include` look fine.